declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="managing-app-sets-in-non-control-plane-namespaces"]
+= Managing the application set resources in non-control plane namespaces
+:context: managing-app-sets-in-non-control-plane-namespaces
+
+toc::[]
+
+:FeatureName: `applicationSet` resources in non-control plane namespaces
+include::snippets/technology-preview.adoc[]
+
+By using application sets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
+
+
+With {gitops-title} 1.12 and later, you can manage the `applicationSet` resources in non-control plane namespaces by explicitly enabling and configuring the Argo CD and `applicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams.
+
+[NOTE]
+====
+The generated Argo CD applications can create resources in any non-control plane namespace. However, the application itself will be in the same namespace as the `applicationSet` resources.
+====
+
+[id="prerequisites_enabling-app-sets-in-any-non-control-plane-namespaces"]
+== Prerequisites
+
+* You have a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance].
+
+* You have link:https://argocd-operator.readthedocs.io/en/latest/usage/apps-in-any-namespace[explicitly enabled and configured] the target namespaces in the Argo CD CR to manage application resources in non-control plane namespaces.
```

Comment:
```suggestion
* You have link:https://argocd-operator.readthedocs.io/en/latest/usage/apps-in-any-namespace[explicitly enabled and configured] the target namespaces in the ArgoCD CR to manage application resources in non-control plane namespaces.
```

Reply:
`ArgoCD` CR

'ApplicationSet` CR vs 1applicationSet` in the spec